deploy/testnet/bin.sh

```diff
@@ -31,7 +31,7 @@ echo "----------------cloning_repository-------------------"
 
 GRAVITY_DIR=$HOME_DIRECTORY/gravity
 ONOMY_DIR=$HOME_DIRECTORY/go/onomy
-git clone https://github.com/onomyprotocol/cosmos-gravity-bridge.git $GRAVITY_DIR
+git clone -b v0.0.0-20210915184851-orch-nomarket https://github.com/onomyprotocol/cosmos-gravity-bridge.git $GRAVITY_DIR
```

Comment:
I don't think it is a good idea to set that branch here because it might be removed. so we should either use commit or just the latest master.

Reply:
I would stick to the master since we have this note that it might be unsafe.

Reply:
Or with the tag